setup.py

```diff
@@ -136,7 +136,7 @@ def run(self):
     long_description=doc(),
     long_description_content_type="text/markdown",
     classifiers=[
-        "Development Status :: 4 - Beta",
+        "Development Status :: 5 - Production/Stable",
```

Comment:
To bych nechal na 1.0.0? Nebo ne?

Reply:
na 1.0.0 čekat nemusíme, ale logo je Beta - nechám Beta.